civis/ml/tests/test_model.py

```diff
@@ -569,7 +569,7 @@ def test_modelpipeline_classmethod_constructor_nonint_id():
 
     out = json.dumps({'job': mp.train_result_.job_id,
                       'run': mp.train_result_.run_id})
-    assert out == '{"job": 1, "run": 2}'
+    assert out == '{"job": 1, "run": 2}' or out == '{"run": 2, "job": 1}'
```

Comment:
I added a comment to this test to explain what's going on. The original error was from `json.dumps` -- we have to serialize the job and run ID to pass them as arguments to the custom script. But if they're an `np.int64` (which you get if you, for example, read them from a CSV with `pandas.read_csv`), then `json.dumps` gets confused and chokes. This test is verifying that we can create the JSON blob.